conf/pacific/cephadm/tier-2_6node_5mons.yaml

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Why do we need this new config ? Can't [1] be reused ?
[1] https://github.com/red-hat-storage/cephci/blob/master/conf/pacific/cephadm/5-node-cluster-multiple-subnet-openstack-only.yaml